apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/AccountControlController.java

```diff
@@ -0,0 +1,462 @@
+            @FormParam("phasingLinkedFullHash") List<String> phasingLinkedFullHash,
+        @Parameter @Schema(description = "phasing 'Hashed Secret' as HEX string", implementation = String.class)
+            @FormParam("phasingHashedSecret") String phasingHashedSecret,
+        @Parameter @Schema(description = "phasing Hashed 'Secret Algorithm' as byte", implementation = Byte.class)
+            @FormParam("phasingHashedSecretAlgorithm") Byte phasingHashedSecretAlgorithm,
+        @Parameter @Schema(description = "recipient Public Key as HEX string", implementation = String.class)
+            @FormParam("recipientPublicKey") String recipientPublicKey,
+        @Parameter @Schema(description = "ec Block Id", implementation = Long.class)
+            @FormParam("ecBlockId") Long ecBlockId,
+        @Parameter @Schema(description = "ec Block Height", implementation = Integer.class)
+            @FormParam("ecBlockHeight") Integer ecBlockHeight,
+
+        @Parameter @Schema(description = "Passphrase to vault account, should be specified if sender account is vault")
+            @FormParam("passphrase") String passphrase,
+        @Parameter @Schema(description = "Two-factor auth code, if 2fa enabled")
+            @FormParam("code2FA") @DefaultValue("0") Integer code2FA,
+        @Context HttpServletRequest servletRequest
+    ) {
+        ResponseBuilder response = ResponseBuilder.startTiming();
+        log.trace("Started leaseBalance, recipientIdParameter = {}, senderIdParameter={}, period={}",
+            recipientIdParameter, senderIdParameter, period);
+
+        long recipientAccountId = recipientIdParameter.get();
+        Account recipientAccount = accountService.getAccount(recipientAccountId);
+        log.trace("recipientAccountId = {}, recipientAccount={}", recipientAccountId, recipientAccount);
+        if (recipientAccount == null || accountService.getPublicKeyByteArray(recipientAccountId) == null) {
+            return response.error(ApiErrors.CUSTOM_ERROR_MESSAGE,
+                "recipient account not found OR it does not have public key").build();
+        }
+        long senderAccountId = senderIdParameter.get();
+        Account senderAccount = accountService.getAccount(senderAccountId);
+        log.trace("senderAccountId = {}, senderAccount={}", senderAccountId, senderAccount);
+
+        Attachment attachment = new AccountControlEffectiveBalanceLeasing(period);
+
+        CreateTransactionRequest txRequest;
+        try {
+            txRequest = HttpRequestToCreateTransactionRequestConverter.convert(
+                servletRequest, senderAccount, recipientAccount, recipientAccountId, 0, feeATM, attachment, broadcast);
+            log.trace("txRequest = {}", txRequest);
+        } catch(ParameterException e) {
+            log.warn("Tx conversion param exception", e);
+            return response.error(ApiErrors.CUSTOM_ERROR_MESSAGE, e.getErrorResponse()).build();
+        } catch(Exception e) {
```

Comment:
Need to catch specific exception here, all other exceptions are treated as `unknown` and are being mapped by DefaultGlobalExceptionMapper

Reply:
Removed try-catch